README.md

```diff
@@ -1,56 +1,29 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Project Portfolio: setup and mobile version skeleton
 
-> Description the project.
+> set up a new repository and prepare it for development using linters.
+build the first 2 sections of the mobile website using the template 1.
 
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
 
+## Built With
 
+- html
+- css
+- linters
+- vscode
 
-## Authors
+## The Milestones:
+- implemented a nav with logo and menu
+- set up a background color and image
+- set up a hader and socials
 
-👤 **Author1**
+👤 **Author**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] Kindly add correct social links to make your documentation more professional. You can add your name at line `21` and social links at each right place. If you don't have one of them I suggest you create or remove it from now.